package.json

```diff
@@ -43,7 +56,7 @@
   },
   "tap": {
     "branches": 85,
-    "functions": 95,
+    "functions": 90,
```

Comment:
Coverage does not appear to be stable. Modifying imports seems to affect coverage, but it's not clear why. We didn't have thresholds before, so dropping to 90% of functions